lib/hocon/impl/simple_config_origin.rb

```diff
@@ -16,7 +16,10 @@ def initialize(description, line_number, end_line_number,
       raise ArgumentError, "description may not be nil"
     end
 
-    @description = description
+    # HACK: naming this variable with an underscore, because the upstream library
+    # has both a member var and a no-arg method by the name "description", and
+    # I don't think Ruby can handle that.
+    @_description = description
```

Comment:
Yeah this part sucks.  I could go the other direction; change the member variable back to `description` and come up with a different name for the method, but in either case I think we are probably going to have to do a sweep looking for all the places that are calling `description` and make sure they're calling the right one.  I'm happy to do it either way.